addons/stock_account/tests/test_account_move.py

```diff
@@ -47,6 +47,24 @@ def setUpClass(cls, chart_template_ref=None):
             }
         )
 
+    @classmethod
+    def create_branch(cls, name=None, parent_company=None):
+        branch = cls.env['res.company'].create({
+            'name': name or 'Branch A',
+            'parent_id': parent_company or cls.env.company.id,
+        })
+        cls.env['account.chart.template'].try_loading('generic_coa', branch, install_demo=False)
+        return branch
+
+    @classmethod
+    def create_comp(cls):
+        comp = cls.env['res.company'].create({
+            'name': 'Company A',
+        })
+
+        cls.env['account.chart.template'].try_loading('generic_coa', comp, install_demo=False)
+        return comp
+
```

Comment:
Can you remove both ?
The branch method is not that useful, the company one is not even used and a better version already exists (`setup_company_data`)

Reply:
I don't even remember why I pushed `create_comp`